src/main/java/frc/trigon/robot/subsystems/intake/IntakeCommands.java

```diff
@@ -38,7 +38,8 @@ private static Command getCollectionCommand() {
                         () -> {
                         },
                         (interrupted) -> {
-                            RobotContainer.INTAKE.sendStaticBrakeRequest();
+                            if (interrupted)
+                                RobotContainer.INTAKE.sendStaticBrakeRequest();
```

Comment:
If it wasn't interrupted, along with indicating collection, we'll start the stop note in place logic (using the brake and everything).
So this should be in the same if as the indication.